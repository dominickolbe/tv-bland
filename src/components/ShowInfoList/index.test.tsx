import { shallow } from "enzyme";
import React from "react";
import tvshowJSON from "../../__mock__/tvshow.json";
import { ListItemRich, ShowInfoList } from "./";

describe("<ListItemRich />", () => {
  it("renders snapshots", () => {
    const wrapper = shallow(
      <ListItemRich primary="Primary text" secondary="Secondary text" />
    );
    expect(wrapper).toMatchSnapshot();
  });
});

describe("<ShowInfoList />", () => {
  it("renders snapshots", () => {
    const wrapper = shallow(<ShowInfoList tvshow={tvshowJSON} />);
    expect(wrapper).toMatchSnapshot();
  });
});
