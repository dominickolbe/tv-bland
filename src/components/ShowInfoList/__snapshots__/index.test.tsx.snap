// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItemRich /> renders snapshots 1`] = `
<Fragment>
  <WithStyles(ForwardRef(ListItem))
    className="makeStyles-listItem-2"
  >
    <WithStyles(ForwardRef(ListItemText))
      className="makeStyles-listItemTextPrimary-3"
      primary={
        <WithStyles(ForwardRef(Typography))
          color="textPrimary"
          component="div"
          variant="body1"
        >
          Primary text
        </WithStyles(ForwardRef(Typography))>
      }
    />
    <WithStyles(ForwardRef(ListItemText))
      className="makeStyles-listItemTextSecondary-4"
      primary={
        <WithStyles(ForwardRef(Typography))
          color="textSecondary"
          component="div"
          variant="body2"
        >
          Secondary text
        </WithStyles(ForwardRef(Typography))>
      }
    />
  </WithStyles(ForwardRef(ListItem))>
  <WithStyles(ForwardRef(Divider))
    component="li"
  />
</Fragment>
`;

exports[`<ShowInfoList /> renders snapshots 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(Grid))
    container={true}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      xs={12}
    >
      <WithStyles(ForwardRef(Typography))
        component="div"
        gutterBottom={true}
        variant="h2"
      >
        <Styled(MuiBox)>
          Show Info
        </Styled(MuiBox)>
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      xs={12}
    >
      <WithStyles(ForwardRef(List))>
        <ListItemRich
          primary="Streamed on"
          secondary="CBS"
        />
        <ListItemRich
          primary="Schedule"
          secondary="Thursday"
        />
        <ListItemRich
          primary="Status"
          secondary="Ended"
        />
        <ListItemRich
          primary="Genre"
          secondary="Comedy"
        />
        <ListItemRich
          primary="Type"
          secondary="Scripted"
        />
        <ListItemRich
          primary="Language"
          secondary="English"
        />
      </WithStyles(ForwardRef(List))>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</div>
`;
